.tx/config

```diff
@@ -1,33 +1,33 @@
 [main]
-host = https://www.transifex.com
+host     = https://www.transifex.com
```

Comment:
Horizontally aligning `=` signs is a waste of time that leads to messy diffs, since adding one key that’s longer than the others involves realigning everything.

Reply:
I don't disagree; this file's modifications were automated with `tx migrate` which chose to align them. I'll take a TODO to undo these in the morning (presuming the `tx` CLI doesn't auto-rewrite the config file on each run or something wild).